site.yml

```diff
@@ -11,6 +11,9 @@ Plugins:
       apiKey: "ALGOLIA_TOKEN"
       index: "INDEX_NAME"
   - Google Analytics: "UA-XXXXX-Y"
+  - Footer Widget:
+      Label: "Talk to community"
+      Link: "/"
```

Comment:
Do we give a link or a token for this integration?

Reply:
@deekoder It's a link.